ipam.go

```diff
@@ -142,6 +159,26 @@ func Half(network net.IPNet) (first, second net.IPNet, err error) {
 	return first, second, nil
 }
 
+// Split returns n subnets from network.
+func Split(network net.IPNet, n uint) ([]net.IPNet, error) {
```

Comment:
While this is used in IPAM as well, this is not IPAM specific. The naming under `ipam` sounds wrong now. :slightly_frowning_face:

Reply:
That is IP address management.